### PR TITLE
Do not spin up N eslint processes (faster lint)

### PR DESCRIPTION
**Pull request by lightsofapollo**
before:

```
PATH=$PATH:$PWD/node_modules/.bin ./test/lint-all.sh  19.84s user 2.28s system 107% cpu 20.555 total
```

after

```
./node_modules/.bin/eslint .  1.29s user 0.09s system 125% cpu 1.101 total
```

### CODE REVIEW

**Comment by dhylands** [comment]
@lightsofapollo Nice

File: package.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
     "passport-local": "^1.0.0",
     "serialport": "^4.0.7",
     "sqlite3": "^3.1.8",
+    "uuid": "^3.1.0",
```
Comment (dhylands) [inline package.json]: This change is probably needed for something else but seems unrelated to this PR

Comment (lightsofapollo) [inline package.json]: oops- this was part of my other PR for the JWT stuff.
